docs/source/posts/2022/2022-08-30-week-11-mohamed.rst

```diff
@@ -0,0 +1,43 @@
+Week 11: Improving tutorials a little
+=====================================
+
+.. post:: August 30 2022
+   :author: Mohamed Abouagour
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+
+This week I didn't do much work.
```

Comment:
good to be honest but not sure we want to read this without explanation. so either explain or remove it.

Reply:
@skoudoro done